faq.adoc

```diff
@@ -28,6 +28,15 @@ So far Kaitai Struct focuses on reading (parsing) only. There are plans to suppo
 
 == How does it compare to ...
 
+=== ... Python library Construct?
+
+Actually, Construct is the closest analog to Kaitai Struct. It is also a symmetric binary parsing library, but the difference is mostly that:
+
+* Construct does both parsing and serialization while Kaitai ([currently](https://github.com/kaitai-io/kaitai_struct/issues/27)) does only parsing
+* Construct is a Python only module while Kaitai supports a multitude of target languages
+
+Main documentation site: https://construct.readthedocs.io
```

Comment:
The rest of these sections use a breakout boxes to present this link, why don't we follow the suit here?

Reply:
Uhm would you want me to add a box? Okies.

Reply:
Preon section doesnt use boxes, and I would rather not do that.